tests/integration/annotation_import/conftest.py

```diff
@@ -291,6 +291,32 @@ def ontology():
     return {"tools": tools, "classifications": classifications}
 
 
+@pytest.fixture
+def wait_for_label_processing():
+    """
+    Do not use. Only for testing.
```

Comment:
it is in the testing dir.... isn't it obvious? :)

Reply:
Copy pasted code from another fixture haha